README.md

```diff
@@ -6,14 +6,38 @@
 
 # caMicroscope distribution
 
-
 run with `docker-compose -f caMicroscope.yml up`
 
 this will build all services and run in the foreground.
 Use `docker-compose -f caMicroscope.yml build` to rebuild the services.
 
 Once everything is up, go to <the host this is running on>:4010/ to see the landing page.
 
+To use docker-compose and run the application, install Docker Desktop Application. <br>
+**NOTE for Windows users:** The Docker Desktop Application also need WSL2 so follow the setup for Windows given below to install Docker on Windows 10. 
+
+**Setup for Windows:**
+1. Install Windows Subsystem for Linux (WSL2) from the link given here: <a href = "https://docs.microsoft.com/en-us/windows/wsl/install-win10#manual-installation-steps">WSL2</a>, **FOLLOW STEPS 1 to 5 ONLY (Step 6 - Install your Linux distribution of choice IS NOT REQUIRED)** 
+2. After completing all the five steps, <a href = "https://hub.docker.com/editions/community/docker-ce-desktop-windows/">Install Docker Desktop for Windows</a>, click on the "Get Docker" button on the right and run the "Docker Desktop Installer" to complete the installation.
```

Comment:
I think this whole section could be replaced by pointing users in the direction of the docker windows installation docs - https://docs.docker.com/docker-for-windows/install/

Reply:
Yes, that can be done but for new contributors, who are also new to docker and wsl it might cause some confusion because the installation docs - https://docs.docker.com/docker-for-windows/install/ is very detailed and technical.